.gitlab/benchmarks/microbenchmarks.yml

```diff
@@ -97,7 +97,7 @@ run-benchmarks:
     # Branch containing 1. scripts to launch Windows benchmarks on ephemeral 
     # instances (to be used by GitLab CI runners) and 2. scripts to run Windows 
     # benchmarks (to be used by the ephemeral instances).
-    BP_INFRA_BENCHMARKING_PLATFORM_BRANCH: "dd-trace-dotnet/micro"
```

Comment:
Guess that this change should be reverted after merging https://github.com/DataDog/benchmarking-platform/pull/215